packages/data-service-generator/src/admin/package-json/create-package-json.ts

```diff
@@ -12,13 +12,12 @@ import { paramCase } from "param-case";
 import { promises as fs } from "fs";
 
 const PACKAGE_JSON_ENCODING = "utf-8";
-const PACKAGE_JSON_TEMPLATE = "package.template.json";
 const PACKAGE_JSON_FILE_NAME = "package.json";
 
-const filePath = resolve(__dirname, PACKAGE_JSON_TEMPLATE);
+const filePath = resolve(__dirname, PACKAGE_JSON_FILE_NAME);
```

Comment:
I am of course not familiar enough with this code, but wouldn't reading a different file (not the .template one) change the functional meaning of this code?

Reply:
We change the `package.template.json` to called `package.json` in order to let the dependabot recognize this file.